pwndbg/color/memory.py

```diff
@@ -51,6 +52,12 @@ def attempt_colorized_symbol(address: int) -> str | None:
     symbol = pwndbg.gdblib.symbol.get(address) or None
     if symbol:
         return get(address, symbol)
+    else:
+        page = pwndbg.gdblib.vmmap.find(address)
+        if page is not None and "[stack" in page.objfile:
```

Comment:
There is `page.is_stack()` fwiw, but it checks for `[stack]` objfile, maybe we should relax this check to look for `[stack` instead -- and then use this method here and elsewhere.

At the same time... maybe we should do `pwndbg.gdblib.vmmap.is_stack(addr)`? cc: @gsingh93  wdyt?